src/Runner/runner.py

```diff
@@ -13,13 +25,17 @@ def run_cpp_code(dest, custom):
         runner = CppRunner(dest, code_file)
 
         if custom:
-            runner.run_on_custom()
+            # runner.run_on_custom()
+            runtime = run_and_get_time(runner, True)
         else :
-            runner.run_on_test_files()
+            # runner.run_on_test_files()
             # validate_output(dest)
+            runtime = run_and_get_time(runner)
             validator = TestValidator(dest)
             validator.validate_output()
         
+        print("Runtime: {} ms".format(runtime))
+        
```

Comment:
Use click.echo instead

Reply:
@akstron Thanks for the feedback. Is it necessary to create a class dedicated towards calculating time, or would it be alright to make changes to the existing PythonRunner, CppRunner and JavaRunner classes themselves?

Reply:
I have modified the PythonRunner, CppRunner and JavaRunner classes by adding a runtime attribute to the runner object, and made changes to the __execute_code method so that it also returns the runtime of the code. This stems from the fact that runtime is naturally a property of the code we are running so it is suitable that we use it as an attribute of our runner object.

Reply:
And I have made the changes to use click.echo() instead of print().